subsys/bluetooth/mesh/Kconfig

```diff
@@ -103,6 +103,64 @@ menuconfig BT_MESH_ADV_EXT
 
 if BT_MESH_ADV_EXT
 
+choice BT_MESH_WORKQ_CONTEXT
+	prompt "Advertising thread selection"
+	default BT_MESH_WORKQ_MESH
+	help
+	  Defines a context for mesh messages transmission.
+
+config BT_MESH_WORKQ_MESH
+	bool "Send messages from in the mesh-specific workqueue"
+	help
+	  When this option is selected, the mesh sends messages from the
+	  mesh-specific workqueue. This will ensure that messages are always sent.
+	  The application needs to ensure the mesh-specific workqueue size is large
+	  enough. Refer to BT_MESH_ADV_STACK_SIZE for the recommended minimum.
+
+if BT_MESH_WORKQ_MESH
+
+config BT_MESH_ADV_STACK_SIZE
```

Comment:
It's confusing to have config options together with choice entries like this, as it makes it a harder for the developer to identify the specific choice entries.

![image](https://github.com/user-attachments/assets/c09d428d-305a-4162-b953-38e4fa3c3300)

Also note, that this construct is only possible because the config types are not bools.
Please move the int entries outside the choice.